test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -0,0 +1,204 @@
+import re
+
+import pytest
+
+from invoke.context import Context
+
+from test.test_utils import LOGGER, ec2
+
+
+def test_stray_files(image):
+    """
+    Test to ensure that unnecessary build artifacts are not present in any easily visible or tmp directories
+
+    :param image: ECR image URI
+    """
+    ctx = Context()
+    container_name = f"test_tmp_dirs-{image.split('/')[-1].replace('.', '-').replace(':', '-')}"
+    _start_container(container_name, image, ctx)
+
+    # Running list of artifacts we do not want in any of the directories
+    stray_artifacts = ["deep-learning-containers.py"]
```

Comment:
Can we investigate more into what other files are typically present in the /tmp folder? Currently we're checking for only this file. Also, a more robust method would be to have 'stray_artifacts', be a list of regex patterns to look for. We'd also need an ignore list to explicitly ignore any known patterns e.g., java artifacts in /tmp when MMS/torchservce start.